src/Test/Annotations.hs

```diff
@@ -69,7 +72,8 @@ testAnnotations setting file rpath = do
                         ,"SRC: " ++ showSrcLoc loc
                         ,"INPUT: " ++ inp
                         ,"OUTPUT: " ++ show i]
-                        | i@Idea{..} <- fromRight [] ideas, let SrcLoc{..} = getPointLoc ideaSpan, srcFilename == "" || srcLine == 0 || srcColumn == 0]
+                        | i@Idea{..} <- fromRight [] ideas, let GHC.SrcLoc{..} = GHC.srcSpanStart ideaSpan, srcFilename == "" || srcLine == 0 || srcColumn == 0]
+                        -- TODO: shouldn't these checks be == -1 instead?
```

Comment:
I'm probably entirely missing something here, but the `== 0`s caught my attention!

Reply:
Think Neil will have to comment on those. Another possibility is that there is some machinery is set-up to inject these spans with those values for the specific reason of triggering those matches.

Reply:
I have no idea. It does seem odd!